ln_wallet_endpoint.py

```diff
@@ -198,14 +197,17 @@ def channel_opener():
             flash(Markup(f'successfully added node id: {node_id[0]} with the amount: {amount}'), 'success')
         except Exception as e: # pylint: disable=broad-except
             flash(Markup(e.args[0]), 'danger')
-    return render_template('lightning/channel_opener.html')
+    rpc = LnRpc()
+    funds_dict=rpc.list_funds()
+    return render_template('lightning/channel_opener.html', funds_dict=funds_dict)
```

Comment:
```suggestion
    return render_template('lightning/channel_opener.html', funds_dict=LnRpc().list_funds())
```

Reply:
removed line 200 and 201. changed line 202 as well.